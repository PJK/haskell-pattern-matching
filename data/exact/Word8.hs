-- Exhaustive, but our solver won't figure that out.
f :: Word8 -> Int
f x | x >= 0 && x < 256 = 1

