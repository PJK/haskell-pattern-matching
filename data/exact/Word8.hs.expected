{
    "tag": "AnalysisSuccess",
    "contents": [
        [
            "f",
            {
                "tag": "NonExhaustive",
                "contents": [
                    [
                        {
                            "tag": "WildcardPattern",
                            "contents": []
                        }
                    ]
                ]
            }
        ]
    ]
}
